utils/s3/credentials_providers/aws_credentials_provider.hh

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2024-present ScyllaDB
```

Comment:
I don't want to spoil the party, but it's 2005 already

Reply:
2005? yoo-hoo! I'm young again!